### PR TITLE
f string in io.common

### PR DESCRIPTION
**Pull request by daniellebrown**
- [ ] ref #29547
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by alimcmaster1** [comment]
Thanks for the PR. Please run black on your changes
```
Checking black formatting
would reformat /home/runner/work/pandas/pandas/pandas/io/common.py
Oh no! 💥 💔 💥
1 file would be reformatted, 875 files would be left unchanged.
```

**Review by alimcmaster1 — CHANGES_REQUESTED** [review]
Please run black

**Review by WillAyd — CHANGES_REQUESTED** [review]
Thanks for the PR. Looks good and I think CI failures are unrelated / have been fixed on master since this was created.

To be sure, can you locally on your shiny-new-feature branch run:

```sh
git fetch upstream
git merge upstream/master
git push origin shiny-new-feature
```

I think should get CI green for this. Flag me down at PyData if you have any questions

W.r.t black you will also want to run `black pandas/io/common.py`

File: pandas/io/common.py [hunk 1]
```diff
@@ -458,12 +458,12 @@ def _get_handle(
...
                     raise ValueError(
-                        "Zero files found in ZIP file {}".format(path_or_buf)
+                        f"Zero files found in ZIP file {path_or_buf}"
                     )
                 else:
                     raise ValueError(
-                        "Multiple files found in ZIP file."
-                        " Only one file per ZIP: {}".format(zip_names)
+                        f"Multiple files found in ZIP file."
+                        " Only one file per ZIP: {zip_names}"
```
Comment (simonjayhawkins) [inline pandas/io/common.py]:
```suggestion
                        f" Only one file per ZIP: {zip_names}"
```
`f` needed on this line

**Comment by alimcmaster1** [comment]
If you merge master as @WillAyd suggested above test failures should be fixed

**Comment by WillAyd** [comment]
I think CI failure is unrelated yet again - can you merge master one more time to be sure?

**Comment by daniellebrown** [comment]
Hi,
I've attempted to merge to master, however, receive the following error messages. Any help much appreciated!

errormsg

**Comment by jreback** [comment]
@daniellebrown you need to ``git push origin shiny-new-feature -f``

**Comment by WillAyd** [comment]
@daniellebrown might want to try one more of:

```
git fetch upstream
git merge upstream/master
git push origin shiny-new-feature
```

To resolve CI failures

**Comment by simonjayhawkins** [comment]
@daniellebrown https://github.com/pandas-dev/pandas/pull/30022#issuecomment-561434493 see https://dev.pandas.io/docs/development/contributing.html#python-pep8-black

**Review by simonjayhawkins — CHANGES_REQUESTED** [review]
https://github.com/pandas-dev/pandas/pull/30022#discussion_r353685117

**Comment by daniellebrown** [comment]
Any advice on how to fix this error: 

would reformat /home/runner/work/pandas/pandas/pandas/io/common.py
Oh no! 💥 💔 💥
1 file would be reformatted, 880 files would be left unchanged.

I've run black several times and it doesn't seem to do the trick.

**Review by simonjayhawkins — APPROVED** [review]
@daniellebrown lgtm @WillAyd

**Comment by jreback** [comment]
thanks @daniellebrown  very nice

**Comment by daniellebrown** [comment]
thanks @jreback @WillAyd @simonjayhawkins !

**Pull request merged**